CMakeLists.txt

```diff
@@ -29,7 +29,14 @@ include(CTest)
 
 cmake_minimum_required(VERSION 2.6)
```

Comment:
Would you consider raising the minimum required version to 3.0?

There's much added goodness that we're missing on like `CMakePackageConfigHelpers` for easy integration in CMake projects.

Reply:
OK, please make a commit or PR